modules/log-upgrade/6x_deleting-red-hat-log-visualization.adoc

```diff
@@ -0,0 +1,42 @@
+:_newdoc-version: 2.18.4
+:_template-generated: 2025-05-20
+:_mod-docs-content-type: PROCEDURE
+
+[id="deleting-red-hat-log-visualization_{context}"]
+= Deleting Red{nbsp}Hat Log Visualization
+
+When updating from Logging 5 to Logging 6, you must delete Red{nbsp}Hat Log Visualization before installing the UIPlugin.
+
+.Prerequisites
+* You have administrator permissions.
+* You installed the {oc-first}.
+
+.Procedure
+. Make each step an instruction.
```

Comment:
In article https://access.redhat.com/articles/7089860 step 5 are described cli steps. Needed to verify by engineering